src/Models/ACF/Composite/CompositeContentFieldGroup.php

```diff
@@ -1451,7 +1486,7 @@ private static function create_acf_field_group()
                                         ],
                                         'return_format' => 'array',
                                         'preview_size' => 'large',
-                                        'library' => 'all',
+                                        'library' => 'uploadedTo',
```

Comment:
Don't change this to 'uploadedTo' - then editors can only select images that are directly uploaded to the current post, and not from the entire library.